lib/sms_text.rb

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
+class Sms_Text
```

Comment:
Style/ClassAndModuleCamelCase: Use CamelCase for classes and modules. (https://github.com/bbatsov/ruby-style-guide#camelcase-classes)